client/my-sites/plugins/plugin-item/plugin-item.jsx

```diff
@@ -34,43 +33,48 @@ function checkPropsChange( nextProps, propArr ) {
 	return false;
 }
 
-module.exports = React.createClass( {
+class PluginItem extends Component {
 
-	displayName: 'PluginItem',
+	constructor( props ) {
+		super( props );
 
-	propTypes: {
-		plugin: React.PropTypes.object,
-		sites: React.PropTypes.array,
-		isSelected: React.PropTypes.bool,
-		isSelectable: React.PropTypes.bool,
-		onClick: React.PropTypes.func,
-		pluginLink: React.PropTypes.string,
-		allowedActions: React.PropTypes.shape( {
-			activation: React.PropTypes.bool,
-			autoupdate: React.PropTypes.bool,
+		this.state = {
+			clicked: false,
+		};
```

Comment:
If you're not going to use the `props` in initializing `state`, you can avoid the `constructor` altogether. For example, https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plugins-wpcom/jetpack-plugins-panel.jsx#L57